flox-file-parser/app/lib/parser.movies.js

```diff
@@ -0,0 +1,108 @@
+const fs = require("fs")
+const path = require("path")
+const db = require("../../database/models")
+
+const videoNameParser = require("video-name-parser")
+
+const supportedVideoFileTypes = ["mkv", "mp4"]
```

Comment:
`supportedVideoFileTypes` is used in `parser.movies.js` and `parser.tv.js`. Maybe move this into an helper to avoid repetition?